generators/entity-client/templates/vue/src/main/webapp/app/entities/entity.vue.ejs

```diff
@@ -166,6 +166,20 @@
                     </td>
                 </tr>
                 </tbody>
+                <%_ if (databaseType !== 'cassandra') { _%>
+                <%_ if (pagination === 'infinite-scroll') { _%>
+                <infinite-loading
+                    ref="infiniteLoading"
+                    v-if="totalItems > itemsPerPage"
+                    :identifier="infiniteId"
+                    slot="append"
+                    @infinite="loadMore"
+                    force-use-infinite-wrapper=".el-table__body-wrapper"
+                    :distance='20'>
+                    <template slot="no-more">No more data</template>
```

Comment:
it could be removed, to be consistent with JHipster+Angular : no need to display this message